members/federal_us_veterans_sec_robert_wilkie.yaml

```diff
@@ -183,11 +183,15 @@ contact_form:
         required: true
     - click_on:
       - selector: button[id*='FormSubmit']
+    - wait:
+      - value: 3
     - find:
       - selector: div.yui3-widget-ft button[class="yui3-button"]
     - click_on:
       - value: Finish Submitting Question
         selector: div.yui3-widget-ft button[class="yui3-button"]
+    - wait:
+      - value: 3
```

Comment:
Can you double up on the `find`s instead of adding hard waits?

Reply:
Updated